pkg/kv/kvserver/kvflowcontrol/rac2/range_controller.go

```diff
@@ -106,14 +114,14 @@ type RaftInterface interface {
 	//
 	// Requires Replica.raftMu to be held, Replica.mu is not held.
 	FollowerStateRaftMuLocked(roachpb.ReplicaID) FollowerStateInfo
-	// SendPingRaftMuLocked sends a MsgApp ping to the given raft peer if there
-	// wasn't a recent MsgApp to this peer. The message is added to raft's message
+	// SendPingLocked sends a MsgApp ping to the given raft peer if there wasn't
+	// a recent MsgApp to this peer. The message is added to raft's message
 	// queue, and will be extracted and sent during the next Ready processing.
 	//
 	// If the peer is not in StateReplicate, this call does nothing.
 	//
-	// Requires Replica.raftMu to be held.
-	SendPingRaftMuLocked(roachpb.ReplicaID) bool
+	// Requires Replica.raftMu and Replica.mu to be held.
```

Comment:
`raftMu` is not required. We only need `mu` so that `RawNode` can safely update its internal state. The fact that `raftMu` is locked is irrelevant/unused here. It's used one level above in `MaybeSendPingsLocked()`.